openc3/lib/openc3/models/interface_model.rb

```diff
@@ -252,9 +253,10 @@ def handle_config(parser, keyword, parameters)
         parser.verify_num_parameters(0, 0, "#{keyword}")
         @log = false
 
-      when 'LOG_RAW'
+      when 'LOG_STREAM', 'LOG_RAW'
+        # TODO: Take parameters for the raw log class? See bridge_config.rb
```

Comment:
This would allow us to change the size but requires specific knowledge of the class initialize parameters. I was thinking something more like the existing `TLM_LOG_CYCLE_TIME` and `TLM_LOG_CYCLE_SIZE` with TLM as STREAM.

Do we instantiate the RawLoggerPair right here like in bridge_config? There's no consistency between the two.